src/components/common/spinner/spinner.scss

```diff
@@ -1,12 +1,23 @@
 @import 'helsinki/colors';
 
 .vene-spinner {
-  border: 2px solid $hel-light;
-  border-top: 2px solid $hel-black;
-  border-radius: 50%;
-  width: 72px;
-  height: 72px;
-  animation: spin 1s linear infinite;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1em;
```

Comment:
Can we use predefined spacing values from here: `src/assets/styles/_variables.scss`?

Reply:
Done.